sec-keys/openpgp-keys-libarchive/openpgp-keys-libarchive-20220324.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="OpenPGP keys used by the libarchive project"
+HOMEPAGE="https://www.libarchive.org/"
```

Comment:
Does the homepage list the key used anywhere?

Reply:
Unfortunately not, I pulled it out of the `.asc`, and the release commits have the same key ID.

Reply:
Guessed so.